packages/EasyErrorHandler/src/Bridge/Laravel/Handler/Handler.php

```diff
@@ -190,7 +190,12 @@ protected function getResponseStatusCode(Exception $exception): int
      */
     protected function isExtendedResponse(): bool
     {
-        return $this->config->get('easy-error-handler.use_extended_response', false) === true;
+        $useExtendedResponse = \filter_var(
+            $this->config->get('easy-error-handler.use_extended_response', false),
+            \FILTER_VALIDATE_BOOLEAN
+        );
+
+        return $useExtendedResponse === true;
```

Comment:
No need for `filter_var` just `(bool)`.

Reply:
Done.

Reply:
Sorry, please, revert to your approach. Because in my way `"false"` will cast to `true`.

Reply:
Done.